index.html

```diff
@@ -414,24 +426,33 @@
       <section data-dfn-for="DiscoveryMethod">
         <h4>The <dfn>DiscoveryMethod</dfn> enumeration</h4>
         <pre class="idl">
-          enum DiscoveryMethod { "any", "local", "nearby", "directory", "broadcast" };
+          enum DiscoveryMethod { "any", "local", "nearby", "directory", "broadcast", "other" };
         </pre>
         <p>
           The <a>DiscoveryMethod</a> enumeration represents the discovery type to be used:
           <ul>
-            <li><dfn>any</dfn> does not provide any restriction</li>
-            <li><dfn>local</dfn> for discovering <a>Things</a> defined in the same device</li>
-            <li><dfn>nearby</dfn> for discovering <a>Things</a> nearby the device, e.g. by Bluetooth or NFC</li>
-            <li><dfn>directory</dfn> for discovery based on a service provided by a directory or repository of <a>Thing</a>s</li>
-            <li><dfn>broadcast</dfn> for an open ended discovery based on sending a request to a broadcast address.</li>
+            <li>"<dfn>any</dfn>" does not provide any restriction</li>
+            <li>"<dfn>local</dfn>" for discovering <a>Things</a> defined in the same device</li>
+            <li>
+              "<dfn>nearby</dfn>" for discovering <a>Things</a> nearby the device, e.g. by Bluetooth or NFC
+            </li>
+            <li>
+              "<dfn>directory</dfn>" for discovery based on a service provided by a directory or repository of <a>Thing</a>s
+            </li>
+            <li>
+              "<dfn>broadcast</dfn>" for an open ended discovery based on sending a request to a broadcast address
+            </li>
+            <li>
+              "<dfn>other</dfn>" for a proprietary method defined by the solution.
+            </li>
           </ul>
         </p>
       </section>
     </section>
 
     <section> <h3>Examples</h3>
       <pre class="example" title="Discover Things via registry">
-        let discoveryType = { type: "registry", url: "http://wot.registry.org" };
+        let discoveryType = { method: "registry", url: "http://wot.registry.org" };
```

Comment:
It can be:

+        let discoveryType = { method: "directory", url: "http://wot.directory.org" };